tide_media.install

```diff
@@ -32,3 +33,31 @@ function tide_media_install() {
       ->save();
   }
 }
+
+/**
+ * SDPA-309: Associated current media items to Creative Commons Attribution 4.0
+ */
+function tide_media_update_8002() {
+  // tide_media_update_8001 is already in usage
+  $termId = null;
+  $terms = \Drupal::entityTypeManager()
+    ->getStorage('taxonomy_term')
+    ->loadByProperties(['name' => 'Creative Commons Attribution 4.0']);
```

Comment:
@enzolutions isn't this supposed to be 'Copyright' instead of Creative Commons?

Reply:
Right now, The option "Creative Commons Attribution 4.0" is the only value, after migrate to use a taxonomy we should retain that option in all content associated, Copyright will be a new option available to choose for Content Editors.